### ISSUE TITLE
/showall 1 and go in spec, hear people double jumping from every distance

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#1532]
As reported by Jesus Christ on discord.

Might be caused by https://github.com/ddnet/ddnet/commit/a7548511bff2b88ed06014fefd7217f8e1b78cc0

**Comment by def-** [comment]
Can't reproduce, tried with Wine and Linux. Anyone who can reproduce it can tell me what's printing when sound is init?

Mine is
```
[2019-03-25 18:57:51][client/sound]: sound init successful using audio driver 'wasapi'
```

**Comment by daverck** [comment]
Can't reproduce on windows 10 pro.
Maybe it's related to the map ?

**Comment by Jupeyy** [comment]
This was probably caused by the bug that was fixed in #4830

If we get new reports we can also create a new issue
